src/DetailsView/components/overview-content/overview-help-section.tsx

```diff
@@ -2,12 +2,11 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { ExternalLinkDeps } from '../../../common/components/external-link';
 import { NamedSFC } from '../../../common/react/named-sfc';
 import { HyperlinkDefinition } from '../../../views/content/content-page';
-import { HelpLinks } from './overview-help-links';
+import { HelpLinks, HelpLinksDeps } from './overview-help-links';
 
-export type HelpLinkDeps = ExternalLinkDeps;
+export type HelpLinkDeps = HelpLinksDeps;
```

Comment:
If this is the deps for `OverviewHelpSection` below, should this be called `OverviewHelpSectionDeps` (and props at line 11 should be `OverviewHelpSectionProps`)?

Reply:
That make sense, except I was not refactoring in a way that I was changing this component directly (and thus, I didn't really touch the left assign here). I understand that we want to go ahead and clean up more things but I would also like for reviewers to see the increased value of applying a PR like this where, yes, you can find related places where you can do more clean up, those places are not in such a good shape -so we want to clean up those places at some point- but are not really being changed on the PR. It seems to this kind of make for bigger than needed PR's and also,  take away part of the speed of using small PR's.

All that being said, I updated this file (and a second one) according to this comments.